From 3b3a7dc2c15cdd30e1ec23205ac2e1b3088258da Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Thu, 27 Aug 2020 08:37:24 +0800
Subject: [PATCH] Test case for Driver - FrontPageStrings.uni file

Description: Change the description in the MdeModulePkg/Application/UiApp/FrontPageStrings.uni

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/Application/UiApp/FrontPageStrings.uni | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdeModulePkg/Application/UiApp/FrontPageStrings.uni b/MdeModulePkg/Application/UiApp/FrontPageStrings.uni
index 8662e009bd..ee6e55af6e 100644
--- a/MdeModulePkg/Application/UiApp/FrontPageStrings.uni
+++ b/MdeModulePkg/Application/UiApp/FrontPageStrings.uni
@@ -11,7 +11,7 @@
 
 #include "JustATest.uni"
 
-#string STR_FRONT_PAGE_TITLE           #language en-US  "Front Page"
+#string STR_FRONT_PAGE_TITLE           #language en-US  "For a test: Front Page"
                                        #language fr-FR  "Front Page"
 #string STR_FRONT_PAGE_COMPUTER_MODEL  #language en-US  ""
                                        #language fr-FR  ""
-- 
2.27.0.windows.1

